inspirehep/modules/forms/validators/simple_fields.py

```diff
@@ -42,16 +39,6 @@ def arxiv_syntax_validation(form, field):
         raise StopValidation(message)
 
 
-def isbn_syntax_validation(form, field):
```

Comment:
This one we might want to leave. As we don't currently have books in the suggestion form but is something that has been asked several times so once we add the `isbn` field we would make use of it. There is an old issue https://github.com/inspirehep/inspire-next/issues/861